lib/Locale/Country/Extra.pm

```diff
@@ -108,241 +118,240 @@ sub _idd_codes {
+        "lv" => 371,
+        "ee" => 372,
+        "md" => 373,
+        "am" => 374,
+        "by" => 375,
+        "ad" => 376,
+        "mc" => 377,
+        "sm" => 378,
+        "va" => 379,
+        "ua" => 380,
+        "rs" => 381,
+        "hr" => 385,
+        "si" => 386,
+        "ba" => 387,
+        "mk" => 389,
+        "it" => 39,
+        "ro" => 40,
+        "li" => 417,
+        "ch" => 41,
+        "cz" => 420,
+        "sk" => 421,
+        "at" => 43,
+        "je" => 441534,
+        "im" => 44,
+        "gb" => 44,
+        "dk" => 45,
+        "se" => 46,
+        "no" => 47,
+        "pl" => 48,
+        "de" => 49,
+        "fk" => 500,
+        "bz" => 501,
+        "gt" => 502,
+        "sv" => 503,
+        "hn" => 504,
+        "ni" => 505,
+        "cr" => 506,
+        "pa" => 507,
+        "pm" => 508,
+        "ht" => 509,
+        "pe" => 51,
+        "mx" => 52,
+        "cu" => 53,
+        "ar" => 54,
+        "br" => 55,
+        "cl" => 56,
+        "co" => 57,
+        "ve" => 58,
+        "gp" => 590,
+        "bo" => 591,
+        "gy" => 592,
+        "ec" => 593,
+        "gf" => 594,
+        "py" => 595,
+        "mq" => 596,
+        "sr" => 597,
+        "uy" => 598,
+        "an" => 599,
+        "my" => 60,
+        "au" => 61,
+        "cx" => 618,
+        "id" => 62,
+        "ph" => 63,
+        "nz" => 64,
+        "pn" => 649,
+        "sg" => 65,
+        "th" => 66,
+        "tl" => 670,
+        "gu" => 671,
+        "aq" => 672,
+        "bn" => 673,
+        "nr" => 674,
+        "pg" => 675,
+        "to" => 676,
+        "sb" => 677,
+        "vu" => 678,
+        "fj" => 679,
+        "pw" => 680,
+        "wf" => 681,
+        "ck" => 682,
+        "nu" => 683,
+        "as" => 684,
+        "ws" => 685,
+        "ki" => 686,
+        "nc" => 687,
+        "tv" => 688,
+        "pf" => 689,
+        "tk" => 690,
+        "fm" => 691,
+        "mh" => 692,
+        "ru" => 7,
+        "kz" => 7,
+        "jp" => 81,
+        "kr" => 82,
+        "vn" => 84,
+        "kp" => 850,
+        "hk" => 852,
+        "mo" => 853,
+        "kh" => 855,
+        "la" => 856,
+        "cn" => 86,
+        "bd" => 880,
+        "tw" => 886,
+        "tr" => 90,
+        "in" => 91,
+        "pk" => 92,
+        "af" => 93,
+        "lk" => 94,
+        "mm" => 95,
+        "mv" => 960,
+        "lb" => 961,
+        "jo" => 962,
+        "sy" => 963,
+        "iq" => 964,
+        "kw" => 965,
+        "sa" => 966,
+        "ye" => 967,
+        "om" => 968,
+        "ps" => 970,
+        "ae" => 971,
+        "il" => 972,
+        "bh" => 973,
+        "qa" => 974,
+        "bt" => 975,
+        "mn" => 976,
+        "np" => 977,
+        "ir" => 98,
+        "tj" => 992,
+        "tm" => 993,
+        "az" => 994,
+        "ge" => 995,
+        "kg" => 996,
+        "uz" => 998,
     };
 
-    my %codes = reverse %$idd_for_codes;
+    my %codes = %$idd_for_codes;
     return \%codes;
 }
```

Comment:
you can just return $idd_for_codes, or even better return { ... } at the top of the sub